azure-monitor-opentelemetry/README.md

```diff
@@ -6,6 +6,8 @@ This distro automatically installs the following libraries:
 
 * [Azure Monitor OpenTelemetry exporters][azure_monitor_opentelemetry_exporters]
 
+as well as a subset of OpenTelemetry [instrumentations][ot_instrumentations] that are officially supported as listed below.
```

Comment:
Since it's a one item list and then a sort of awkward "as well" could we could make these 3 lines one paragraph?